src/docs/01-get-started/index.md

```diff
@@ -6,15 +6,10 @@ permalink: /docs/get-started/
 ---
 # Overview
 
-A large number of use cases span beyond a single request-reply, require tracking
-of a complex state, respond to asynchronous :event:events:, and communicate to external unreliable dependencies.
-The usual approach to building such applications is a hodgepodge of stateless services,
-databases, cron jobs, and queuing systems. This negatively impacts the developer productivity as most of the code is
-dedicated to plumbing, obscuring the actual business logic behind a myriad of low-level details. Such systems frequently have availability problems as it is hard to keep all the components healthy.
-
-The Cadence solution is a [_fault-oblivious stateful_ programming model](/docs/concepts/workflows) that obscures most of the complexities of building scalable distributed applications. In essence, Cadence provides a durable virtual memory that is not
-linked to a specific process, and preserves the full application state, including function stacks, with local variables across all sorts of host and software failures.
-This allows you to write code using the full power of a programming language while Cadence takes care of durability, availability, and scalability of the application.
+As more and more tenants adopt a microservice-oriented architecture to meet their business use cases, managing a large-scale microservice architecture is not a simple task. Engineers not only need to develop business features but also have to enforce substantial maintenance and observability throughout the software development and deployment lifecycle. For example, managing an end-to-end workflow state machine may involve managing tasks such as asynchronous events, service-to-service communications, recovery from faults and disasters, database failovers, and cron jobs, among others. All of these components consume a considerable amount of engineering resources and can eventually slow business growth. Therefore, engineers are eager to have a managed solution to help them focus on business feature inquiries and reduce operational overhead.
+
+
+The motivation behind the Cadence solution is to provide service producers with an easy-to-use, fault-oblivious stateful workflow model. It abstracts the common complexities of building modern scalable distributed applications and provides a managed platform for customers to develop and maintain their services. What engineers do with Cadence is simply write code for critical business logic, and Cadence will manage durability, availability, and scalability for them. This approach helps businesses grow without accumulating overwhelming operational backlogs.
```

Comment:
on a technical side, "scalability" is kinda questionable since most of the behavior is run by their service.... but yea, I think that's fine.

operational backlogs I'm a bit more fuzzy on.  at the core, cadence is a complicated series of queue processors - it has task-backlogs _by design_.  "operational" could be referring to business processes, and it does probably help there because it's a good coordinator... but it feels kinda unclear if it's referring to that or "during operation, you will not accumulate overwhelming backlogs", and the latter is definitely not true.